app/models/changelog_data.php

```diff
@@ -1,16 +1,19 @@
 <?php
 
 $author_urls = [
-    'flod'       => 'https://github.com/flodolo',
-    'geb'        => 'https://github.com/mgoessen',
-    'jesus'      => 'https://github.com/tx2z',
-    'keko'       => 'https://github.com/keko',
-    'MikkCZ'     => 'https://github.com/MikkCZ',
-    'pascal'     => 'https://github.com/pascalchevrel',
-    'SkySymbol'  => 'https://github.com/SkySymbol',
-    'sphinx'     => 'https://github.com/SphinxKnight',
-    'tchevalier' => 'https://github.com/TheoChevalier',
-    'Thegennok'  => 'https://github.com/Thegennok',
+    'Benoit'      => 'https://github.com/benoit-l',
+    'Clement'     => 'https://github.com/clement-lefevre',
+    'flod'        => 'https://github.com/flodolo',
+    'geb'         => 'https://github.com/mgoessen',
+    'jesus'       => 'https://github.com/tx2z',
+    'keko'        => 'https://github.com/keko',
+    'Anthony'     => 'https://github.com/MatonAnthony',
```

Comment:
The order of this array is pretty weird, should we order them by displayed name, given it's also easy to do it directly in Atom?